packages/main/src/themes/ProductSwitch.css

```diff
@@ -0,0 +1,31 @@
+:host {
+	font-family: var(--sapUiFontFamily);
+	font-size: var(--sapMFontMediumSize);
+}
+
+.ui5-productswitch-root {
+	display: flex;
+	flex-wrap: wrap;
+	width: 778px;
+	padding: 1.25rem 0.75rem;
+	box-sizing: border-box;
+}
+
+:host([desktop-columns="3"]) .ui5-productswitch-root {
+	width: 588px;
+}
+
+@media only screen and (max-width: 900px) {
```

Comment:
we can not assume that the parent component (could be the shellbar) is always on root level. Therefore I propose the responsive API instead of hardcoding widths for different sizes of the screen